SU2_PY/SU2/io/config.py

```diff
@@ -85,9 +85,12 @@ def __init__(self,*args,**kwarg):
         # read config if it exists
         if filename:
             try:
-                self.read(filename)
-            except:
-                raise IOError , 'Could not find config file: %s' % filename
+                self.read(filename)	# is really read with IO Exceptions, and process with Asserts
+            except IOError:
+                print 'Could not find config file: %s' % filename
+	    except:			# for asserts that test validity of user modified config file
```

Comment:
this block isn't aligning with the rest of the statement. make sure you're using spaces too!